src/lib/Menu.js

```diff
@@ -109,17 +125,25 @@ const Menu = (props) => (
     <NavMenu>
       {props.links.map((item, index) =>
         item.submenu ? (
-            <SubbedNavItem key={item.name + index}>
-              <NavLink href={item.link}>
+            <SubbedNavItem
+              key={item.name + index}
+              hoverLine={props.hoverLine}
+              hasSubMenu={true}
```

Comment:
I don't see `SubbedNavItem` using `hasSubMenu`

Reply:
`SubbedNavItem` extends `NavItem` which uses

Reply:
Ok, didn't notice it :+1: